packages/starrating/src/react/half-star-dark.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+
+const DarkHalfStar = () => (
+  <svg
+    width="24"
```

Comment:
We usually try to avoid styling and static dimensions in svg.  Rely on viewBox for dimensions and css for the styling.